intern/service/boss/baseversion.py

```diff
@@ -15,7 +15,12 @@
 from abc import ABCMeta, abstractmethod
 from intern.resource.boss.resource import CoordinateFrameResource
 from requests import Request
+from enum import Enum
 
+class CacheMode(str, Enum):
```

Comment:
I don't think this should have been moved here to make the unit tests pass. The unit tests should be modified to accommodate the proper location of the new enum.